stdlib/doctest.pyi

```diff
@@ -41,9 +42,22 @@ __all__ = [
     "debug",
 ]
 
-class TestResults(NamedTuple):
-    failed: int
-    attempted: int
+# MyPy errors on conditionals within named tuples.
+
+if sys.version_info >= (3, 13):
+    class TestResults(NamedTuple):
```

Comment:
The extra keyword argument in `__new__` breaks the inference for `_fields` and `__match_args__` - even when overriding them with a type ignore stubtest still things these `__match_args__` and `_fields` are  `skipped`, `failed`, and `attempted` and raises an error since that doesn't match the runtime.